bytecode/concrete.py

```diff
@@ -280,11 +280,7 @@ def _assemble_lnotab(first_lineno, linenos):
         return b"".join(lnotab)
 
     @staticmethod
-    def _pack_linetable(doff, dlineno):
-        linetable = []
-        while doff > 254:
-            linetable.append(b"\xfe\x00")
-            doff -= 254
+    def _pack_linetable(doff, dlineno, linetable):
 
         while dlineno < -127:
```

Comment:
Line may be too long with comment...

```suggestion
        # Ensure linenos are between -126 and +126, by using 127 lines jumps with a 0 byte offset
        while dlineno < -127:
```